tezos/interop/lib_tezos/libtezos-ffi-distribution-summary.json

```diff
@@ -1,481 +1,481 @@
 [
```

Comment:
@bruno.deferrari 
wow, you removed all the backtrack/recording stuff from apply_block in ffi, very very very good :)
just please, squash commits and merge to tezedge-ffi with tag `v1.1.1`  with new libtezos-ffi-distribution,
because I need to tomorrow fix that edo stuff in ffi